fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluids/v1/container/volume/FluidVolume.java

```diff
@@ -0,0 +1,45 @@
+package net.fabricmc.fabric.api.fluids.v1.container.volume;
+
+import java.util.Iterator;
+
+import com.google.common.collect.Iterators;
+
+import net.minecraft.fluid.Fluid;
+import net.minecraft.nbt.CompoundTag;
+
+import net.fabricmc.fabric.api.fluids.v1.container.FluidContainer;
+
+/**
+ * a fluid container that can only hold 1 fluid.
+ */
+public interface FluidVolume extends FluidContainer {
```

Comment:
If this can only hold 1, should this be called something related to SimpleFluidVolume

Reply:
like what, Simple_FluidVolume_ seems related enough to me